tests/core/editable/keystrokes/delbackspacequirks/manual/nbspbackspacing.html

```diff
@@ -0,0 +1,11 @@
+<textarea id="editor" cols="30" rows="10">
+	<p>Hello World!</p>
+</textarea>
+
```

Comment:
Let's add a paragraph below the editor tracking the editor source code on `change` event to improve debugging experience.